docs/best_practices.md

```diff
@@ -189,6 +189,10 @@ Here are some great examples of what tests and test coverage should look like.
 - [RegisterButton.jest.tsx](https://github.com/artsy/force/blob/main/src/v2/Apps/Auction/Components/__tests__/RegisterButton.jest.tsx)
 - [ResetPasswordRoute.jest.tsx](https://github.com/artsy/force/blob/main/src/v2/Apps/Authentication/Routes/__tests__/ResetPasswordRoute.jest.tsx#L8)
 
+#### Manipulating time in unit tests
```

Comment:
@jacobherrington - mind adding a link to this in the table of contents up above?

Reply:
I made it a subheading because I felt it wasn't large enough for a section, but sure. :)